tools/testbin

```diff
@@ -111,29 +111,40 @@ finish_test() {
     stop_postgres
 }
 
-test_deb() {
+deb_init() {
     [ -n "$OS_NAME" ] || die 'OS_NAME environment variable must be set to the distribution name e.g. debian or ubuntu'
     [ -n "$OS_VERSION" ] || die 'OS_VERSION environment variable must be set to the distribution version number'
 
     ARCH=`dpkg --print-architecture`
     EPOCH=
     MIN_DEB_ARM=`cmp_version $MIN_DEB_ARM`
     MIN_DEB_EPOCH=`cmp_version $MIN_DEB_EPOCH`
-    for from_version; do
+}
+
+# Requires:
+# - FROM_VERSION
+deb_start_test() {
         # We released 1.10.0-dev by accident.  We have to support upgrades
         # from it (and we tested that at the time), but we pulled the deb, so
         # we can't test it here.
-        [ $from_version = 1.10.0-dev ] && continue
-        cmp_version=`cmp_version $from_version`
-        [ "$ARCH" -eq arm64 ] && [ $cmp_version -lt $MIN_DEB_ARM ] && continue
+        [ $FROM_VERSION = 1.10.0-dev ] && return 1
+        cmp_version=`cmp_version $FROM_VERSION`
+        [ "$ARCH" -eq arm64 ] && [ $cmp_version -lt $MIN_DEB_ARM ] && return 1
+
         [ $cmp_version -ge $MIN_DEB_EPOCH ] && EPOCH=1:
         for PG_VERSION in $PG_VERSIONS; do
             select_pg $PG_VERSION
-            deb=timescaledb-toolkit-postgresql-${PG_VERSION}=${EPOCH}${from_version}~${OS_NAME}${OS_VERSION}
+            deb=timescaledb-toolkit-postgresql-${PG_VERSION}=${EPOCH}${FROM_VERSION}~${OS_NAME}${OS_VERSION}
             $nop sudo apt-get -qq install $deb
 
             PATH=/usr/lib/postgresql/$PG_VERSION/bin:$PATH start_test
         done
```

Comment:
This section is indented twice; it should be indented once.

Reply:
I left that as is for clear diff on first review, but then forgot!  Fixed now.  Thanks.